packages/opentelemetry-browser-extension-autoinjection/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "@opentelemetry/browser-extension-autoinjection",
+  "version": "0.21.0",
+  "description": "A browser extension that injects opentelemetry tracers into any website",
+  "scripts": {
+    "clean": "rimraf build/*",
+    "compile": "tsc --build tsconfig.json",
+    "build": "npx webpack --mode=production",
+    "build:mv2": "npx webpack --mode=production --env MV=2",
+    "build:mv3": "npx webpack --mode=production --env MV=3",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "precompile": "tsc --version",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "watch": "npx webpack --mode=development --watch",
+    "watch:mv2": "npx webpack --mode=development --watch --env MV=2",
+    "watch:mv3": "npx webpack --mode=development --watch --env MV=3"
+  },
+  "private": true,
+  "keywords": [],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "@opentelemetry/api": "^0.21.0"
+  },
+  "devDependencies": {
+    "@opentelemetry/api": "0.21.0",
```

Comment:
i think we can directly use api 1.0 (PR to update all contrib is there: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/540)

Reply:
done

Reply:
@vmarchaud this does not work with Node.JS 16 as it seems. I am not sure yet how to fix this since it works locally for me:

```Run npx lerna bootstrap --no-ci
lerna notice cli v3.22.1
lerna info Bootstrapping 33 packages
lerna info Installing external dependencies
lerna ERR! npm install exited 1 in '@opentelemetry/browser-extension-autoinjection'
lerna ERR! npm install stderr:
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: @opentelemetry/browser-extension-autoinjection@0.21.0
npm ERR! Found: @opentelemetry/api@1.0.0
npm ERR! node_modules/@opentelemetry/api
npm ERR!   dev @opentelemetry/api@"1.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @opentelemetry/api@"^0.21.0" from @opentelemetry/core@0.21.0
npm ERR! node_modules/@opentelemetry/core
npm ERR!   @opentelemetry/core@"^0.21.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /github/home/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /github/home/.npm/_logs/2021-06-21T09_08_46_212Z-debug.log

lerna ERR! npm install exited 1 in '@opentelemetry/browser-extension-autoinjection'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
Error: Process completed with exit code 1.```

Reply:
Okay it seems we'll need to merge #540 before, since core packages peer depend on 0.12 for contrib we can't bump (i think its because of a new npm version)

Reply:
@svrnm Should be good now #540 has been merged, could you try again ?

Reply:
Sure! Let's see if that works :-)

Reply:
@vmarchaud ... no, same error :-(